introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi, my name is Nicolò Scarpa and i'm Happy to be introduced to the dojo.
 My favorite Code School path is JavaScript.
+My deadly skills are: 
+* PHP
+* MVC design
```

Comment:
Even if i'm not a real "master" in this skill, i will add it just because you asked me.